dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -509,13 +509,13 @@ def get_airflow_extras():
     {
         "python-version": "3.8",
         "airflow-version": "2.8.4",
-        "remove-providers": "cloudant fab",
+        "remove-providers": ["fab", "standard"],
```

Comment:
I think we should aim to get standard to be **not** removed. Not sure why we are removing it now - but we agreed that 2.8 should be the "min-version" for standard provider, so the tests should be adapted to work for those versions as well..

Reply:
I remember why I did it and kept the min version to be 2.10.
We talked about it in slack: https://apache-airflow.slack.com/archives/C06K9Q5G2UA/p1724737809230079